initrd/bin/unseal-totp

```diff
@@ -8,14 +8,14 @@ TOTP_SECRET="/tmp/secret/totp.key"
 TRACE "Under /bin/unseal-totp"
 
 if [ "$CONFIG_TPM" = "y" ]; then
-	tpmr unseal 4d47 0,1,2,3,4,7 312 "$TOTP_SECRET" \
-		|| die "Unable to unseal totp secret"
-fi
+	tpmr unseal 4d47 0,1,2,3,4,7 312 "$TOTP_SECRET" ||
+		die "Unable to unseal TOTP secret"
+		fi
```

Comment:
Whoops this got mis-indented, could you fix that please?

Reply:
@JonathonHall-Purism fixed under [9523b4f](https://github.com/linuxboot/heads/pull/1515/commits/9523b4fee22f936f940198fee25ab67af401e180)